libs/common/src/assets/i18n/en.json

```diff
@@ -828,6 +828,8 @@
 	"resource.origin.related": "Related",
 	"resource.origin.url": "URL",
 	"resource.pagination.counter": "{{from}}-{{to}} of {{total}}",
+	"resource.pagination.download": "Download list",
```

Comment:
I think we must be more specific, like "Download the entire list", because with "Download list", the user might think it will download only what is currently filtered.

Reply:
Fixed